packages/db-objects-redis/src/lib/objects/objectsInRedisClient.ts

```diff
@@ -3255,11 +3259,11 @@ export class ObjectsInRedisClient {
         return this.getObjects(keys, options);
     }
 
-    async _getObjectsByPattern(
+    private async _getObjectsByPattern(
         pattern: string,
         options: CallOptions,
         callback: (err?: Error | null, objs?: ioBroker.Object[]) => void
-    ) {
+    ): Promise<ioBroker.Object[] | void> {
```

Comment:
Depending on how strict the calling code should be, maybe `ioBroker.AnyObject` is a better fit for some locations where `ioBroker.Object` is used currently. The main difference is that `ioBroker.Object` doesn't care about whats in `common` and `native`:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/06f38fe1967d69ed164552b55674bf2d5269e7a1/types/iobroker/objects.d.ts#L653-L666
vs
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/06f38fe1967d69ed164552b55674bf2d5269e7a1/types/iobroker/objects.d.ts#L686-L691